.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
Sorry @sokra, but it's not clear to me. Can you elaborate your question, please?

This configuration is only for this repository, so the user that is consuming/using the package can use other configuration.